README.md

```diff
@@ -69,5 +69,29 @@ A small example how this library is efficient in reading a `json` file _(more pa
   ]);;    
 - : (Topojson.t, [ `Msg of string ]) result = Ok <abstr>
 ```
+### Reading a raw JSON string
 
+```ocaml
+
+# let topojson_string = {|{
+    "arcs": [[[0.0, 0.0], [0.0, 9999.0], [2000.0, 0.0], [0.0, -9999.0], [-2000.0, 0.0]]],
+    "objects": {"example ": {
+            "type": "GeometryCollection",
+            "geometries": [
+                {"coordinates": [4000.0, 5000.0],
+                 "properties": {"prop0": "value0"},
+                 "type": "Point"},
+                {"arcs": [[0]],
+                 "properties": {"prop0": "value0", "prop1": {"this": "that"}},
+                 "type": "Polygon"}
+            ]
+     }}
```

Comment:
```suggestion
     }|}
```

You need to terminate the string as the compiler error is pointing out. Once you have that working you can then add the actual reading of the string.